### PR TITLE
Add a codeowners file and PR Template

### PR DESCRIPTION
**Pull request by astronaut-wannabe**
# How does this PR make you feel (in animated GIF format)?
extra

# What are the relevant GH issues?
n/a

# What's this PR do?
## Codeowners File
This will add a check before any code gets merged in that modifies the
db schema or the export rake task

I left the examples from the [github docs][0] in the PR for now in
case anything else comes up during the PR process, but I will remove
all of the commented out examples before merging the PR

## PR Template
I am not super tied to this template or anything, but I did want an
extra reminder for PRs that mentioned the db export script again

# How should this be manually tested?
- open a PR 
  - should see the template populate the PR description
- modify the files covered by the new CODEOWNERS file
  - should see @veganstraightedge auto-assigned as a reviewer
# Any background context you want to provide?
We want to make sure the new db export rake task doesn't get out of
sync with the db/schema (and possibly leaking sensitive info)

We opted for this approach (as opposed to something like securing db
dumps behind ssh keys) in order to keep friction low for people who
would like to jump in and help.

# Acceptance Criteria:
## These should be checked by the reviewer(s).
- [X] Does not cause the db export script to become out of sync with the db schema

[0]:https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# Migrations must be approved in order to make sure the db export rake
+# task is still scrubbing sensitive info
+/lib/tasks/db/ @veganstraightedge
+/db/ @veganstraightedge
```
Comment (astronaut-wannabe) [inline .github/CODEOWNERS]: this is the only "real" config in this file.

I left the other examples below just for the PR process, but will delete before merging

Comment (veganstraightedge) [reply .github/CODEOWNERS]: Righteous. IMO, you should add yourself to this list too.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# Migrations must be approved in order to make sure the db export rake
```
Comment (veganstraightedge) [inline .github/CODEOWNERS]:
```suggestion
# Migrations must be approved in order to make sure the database export rake
```

File: .github/PULL_REQUEST_TEMPLATE.MD [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# How should this be manually tested?
+
+??
+
+# Is there any background context you want to provide for reviewers?
+
+??
+
+# Questions for pull requests author (delete any that don't apply):
```
Comment (astronaut-wannabe) [inline .github/PULL_REQUEST_TEMPLATE.MD]:
```suggestion
# Questions for the pull request author (delete any that don't apply):
```

**Pull request merged**